packages/replay/src/eventBuffer.ts

```diff
@@ -160,6 +175,9 @@ export class EventBufferCompressionWorker implements EventBuffer {
     });
   }
 
+  /**
+   *
+   */
   private _sendEventToWorker(event: RecordingEvent): Promise<ReplayRecordingData> {
```

Comment:
l: Do private methods need a JSDoc? I always thought this was just a requirement for public/exported methods/functions? I'm not saying, private methods shouldn't have one btw 😅  just that adding empty stubs seems weird to me (unless the lint rules enforce this ofc).

Reply:
ah, this empty stub I missed! Yeah, I was also surprised, but eslint made me do it xD As far as I see, private methods need jsdoc, but private properties don't.